Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings

```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "Kurzbefehl aufnehmen";
+"press_shortcut" = "Kurzbefehl wählen...";
+"keyboard_shortcut_used_by_menu_item" = "Diese Tastaturkürzel kann nicht verwendet werden, da es bereits durch den Menüpunkt „%@” belegt ist.";
+"keyboard_shortcut_used_by_system" = "Dieses Tastaturkürzel kann nicht verwendet werden, da es systemweit bereits verwendet wird.";
+"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können in  „Systemeinstellungen › Tastatur › Kurzbefehle“.";
```

Comment:
```suggestion
"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können unter  „Systemeinstellungen › Tastatur › Tastaturkurzbefehle“ geändert werden.";
```

Reply:
There is one space too much before "„Systemeinstellungen >

Reply:
> "Die meisten systemweiten Tastaturkürzel können unter „Systemeinstellungen › Tastatur › Tastaturkurzbefehle“ geändert werden."

```suggestion
"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können unter  „Systemeinstellungen › Tastatur › Kurzbefehle“ geändert werden.";
```

Reply:
```suggestion
"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können unter  „Systemeinstellungen › Tastatur › Kurzbefehle“ geändert werden.";
```